controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go

```diff
@@ -30,6 +31,20 @@ func (r *ClusterReconciler) ReconcileAddonSecret(
 		log.Info("Failed to get cluster avi user secret, requeue")
 		return res, err
 	}
+
+	// when avi is ha provider and deploy ako in management cluster, need to wait for
+	// control plane load balancer type of service creating
+	if akoo.IsHAProvider() && cluster.Namespace == akoov1alpha1.TKGSystemNamespace {
+		svc := &corev1.Service{}
```

Comment:
If ako addon secret not generated, AKO is not started?

Reply:
Yes, without the secret, it won't deploy AKO